server/index.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../database/index.js')
+const express = require('express')
+const path = require('path')
+const app = express()
+const port = 3000
+const seed = require('../database/seed.js')
```

Comment:
you may want to look into using the Airbnb styleguide w/ eslint. it will provide many suggestions to make your code more readable, such as in the way to order required files